modules/nodes-scheduler-default-sample.adoc

```diff
@@ -10,19 +10,32 @@ were to be specified using the scheduler policy file.
 
 [source,yaml]
 ----
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: mypolicy
-  namespace: openshift-config
-  selfLink: /api/v1/namespaces/openshift-config/configmaps/mypolicy
-  uid: 83917dfb-4422-11e9-b2c9-0a5e37b2b12e
-  resourceVersion: '1100851'
-  creationTimestamp: '2019-03-11T17:24:23Z'
-data:
-  policy.cfg: "{\n\"kind\" : \"Policy\",\n\"apiVersion\" : \"v1\",\n\"predicates\" : [\n\t{\"name\" : \"PodFitsHostPorts\"},\n\t{\"name\" : \"PodFitsResources\"},\n\t{\"name\" : \"NoDiskConflict\"},\n\t{\"name\" : \"NoVolumeZoneConflict\"},\n\t{\"name\" : \"MatchNodeSelector\"},\n\t{\"name\" : \"HostName\"}\n\t],\n\"priorities\" : [\n\t{\"name\" : \"LeastRequestedPriority\", \"weight\" : 10},\n\t{\"name\" : \"BalancedResourceAllocation\", \"weight\" : 1},\n\t{\"name\" : \"ServiceSpreadingPriority\", \"weight\" : 1},\n\t{\"name\" : \"EqualPriority\", \"weight\" : 1},\n\t{\"name\" : \"labelPreference\", \"label\" : \"rack\", \"presence\" : \"true\", \"weight\" : 2},\n\t]\n}\n "
+kind: "Policy"
+version: "v1"
+predicates:
+...
+  - name: "RegionZoneAffinity" <1>
+    argument:
+      serviceAffinity: <2>
+        labels: <3>
+          - "region"
+          - "zone"
+priorities:
+...
+  - name: "RackSpread" <4> 
+    weight: 1
+    argument:
+      serviceAntiAffinity: <5> 
+        label: "rack" <6>
```

Comment:
@anpingli Can i bother you to look at these example policy.cfg files? I tried two and am getting CrashLoopBackOff errors. Something wrong with the formatting of these files?
_invalid policy: couldn't get version/kind; json parse error: invalid character 'k' looking for beginning of object key string_